src/style.css

```
@@ -0,0 +1,25 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+.head {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.Subtitle {
```

```suggestion
.subtitle {
```
since all other classes are lowercased